backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/ZipApi.java

```diff
@@ -59,16 +59,15 @@ static String getZip(Request request, Response response) throws IOException {
     tempDir.toFile().deleteOnExit();
     try (OutputStream outputStream = response.raw().getOutputStream()) {
       Schema schema = getSchema(request);
+      String fileName = schema.getMetadata().getName() + System.currentTimeMillis() + ".zip";
+      // First set headers before stream blob.
```

Comment:
Comment should be removed, as it adds no information. Or it should explain why headers need to be added before the blob.